src/sentry/newsletter/remote.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
+
+import logging
+from uuid import uuid4
+from hashlib import sha1
+
+from sentry.http import safe_urlopen
+from sentry.newsletter.base import Newsletter
+from sentry.utils import json
+
+logger = logging.getLogger('sentry.newsletter')
+
+
+def get_install_id():
```

Comment:
@dcramer no, it's "enforced" during beacon checkin. But yeah, I'll follow up in a different PR to just enforce install id to exist, and centralize this logic since this was copy/pasted from the beacon code.